db/migrate/20230404052334_irreversable_migration_example.rb

```diff
@@ -0,0 +1,9 @@
+class IrreversableMigrationExample < ActiveRecord::Migration[7.0]
+  def change
+    create_table :articles do |t|
+      t.string :name
+      t.boolean :is_public?
+    end
+    change_column_default :articles , :is_public? , true
```

Comment:
spacing issue
how is this example of irreversible migration?

Reply:
because I don't provide from and to so :is_public never knows its previous state.